libs/wingsdk/src/target-sim/function.inflight.ts

```diff
@@ -75,7 +77,7 @@ export class Function implements IFunctionClient, ISimulatorResourceInstance {
     return this.context.withTrace({
       message: `Invoke (payload="${JSON.stringify(payload)}").`,
       activity: async () => {
-        return vm.runInContext(wrapper, context);
+        return vm.runInContext(wrapper, context, { timeout: this.timeout });
```

Comment:
Aha, so there's a timeout option! Brilliant :)

Reply:
yes, I thought that would be clean solution, to make sure the script execution is really stopped.